actionlib/test/simple_client_test.cpp

```diff
@@ -104,18 +98,21 @@ TEST(SimpleClient, easy_callback)
   bool started = client.waitForServer(ros::Duration(10.0));
   ASSERT_TRUE(started);
 
+  // sleep a bit to make sure that all topics are properly connected to the server.
+  ros::Duration(0.01).sleep();
```

Comment:
i believe this should be done by waitForServer.

Reply:
Yep, it should. But actually it isn't, see my comment above: https://github.com/ros/actionlib/pull/156#issuecomment-595713514

Reply:
i understand this is just a work-around to pass the test, but even with this duration SimpleClientFixture fails with server state is still ACTIVE (expected to be SUCCESS). so I expect this additional duration is not needed.

Reply:
It's the unrelated test file `exercise_simple_client.cpp` that fails - adding another sleep over there would help.
```
[actionlib.rosunit-test_cpp_simple_client/just_succeed][FAILURE]
 /tmp/ws/src/actionlib/actionlib/test/exercise_simple_client.cpp:72
```